Framework/Algorithms/src/DiffractionFocussing2.cpp

```diff
@@ -109,6 +109,11 @@ void DiffractionFocussing2::exec() {
     g_log.error() << "UnitID " << unitid << " is not a supported spacing\n";
     throw std::invalid_argument("Workspace Invalid Spacing/UnitID");
   }
+  if (unitid == "TOF") {
+    g_log.error()
```

Comment:
I think if we leave this at error level the red text in the GUI will make people think that there has been an error. Can we put this at `warning`?

Reply:
I think Richard originally had this as a warning and I suggested he change it to error because that's the level that the "deprecated algorithm" messages get raised at. Fine with me to change back though if you prefer warning

Reply:
Ah yes good point. For reason I expected those would be at warning level too. I think you're right that the consistency here is better. Thanks!